Apps/Common/test/unit/Services/SupportServiceTests.cs

```diff
@@ -187,6 +196,50 @@ public void ShouldGetVerifications()
             Assert.Equal(2, actualResult.ResourcePayload.Count());
         }
 
+        /// <summary>
+        /// Tests SupportService can get a list of delegate owner by the delegate's PHN.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Fact]
+        public async Task ShouldGetResourceDelegateOwners()
```

Comment:
This test looks like it's using inverted logic for delegate and dependent.

Reply:
fixed, please re-review